### ISSUE TITLE
req.engine.queue() doesn't support the "label" keyword

### ISSUE THREAD

**Issue by ngregoire** [PortSwigger/turbo-intruder#66]
In queueRequest(), using the _label_ keyword is accepted:
`engine.queue(target.req, "", label='x')`

But not in handleResponse():
`req.engine.queue(req.template, "something"), label='z')`

The error message:
`Error in user-defined callback: Traceback (most recent call last):
  File "<string>", line 26, in handleResponse
TypeError: queue(): takes no keyword arguments`

Tested on Burp v2021.2.1 + Turbo Intruder v1.1.3

**Comment by ngregoire** [comment]
Note: using `engine.queue()` with the _label_ keyword works from handleResponse() if the `engine` created in queueRequest() is exposed using a global variable

**Comment by albinowax** [comment]
Yeah, req.engine is a reference to the Java class so it misses out on the features offered by the Python wrapper. This is a known issue but it's on my todo list.
